docs/meshstack.azure.landing-zones.md

```diff
@@ -3,18 +3,15 @@ id: meshstack.azure.landing-zones
 title: Landing Zones
 ---
 
-In Azure, a [Landing Zone](./meshcloud.landing-zones.md) is defined via a management group the subscription for the project will be assigned to. Policies can be applied
-to these management groups. Optionally a blueprint can also be defined. Via an Azure Blueprint default resources can be deployed to the
-subscription and additional specific policies can be defined. A blueprint can be configured to decline users to change or delete the
-resources and policies created by the blueprint.
+In Azure, a [Landing Zone](./meshcloud.landing-zones.md) is defined via a management group the subscription for the project will be assigned to. Policies can be applied to these management groups. Optionally a blueprint can also be defined. Via an Azure Blueprint default resources can be deployed to the subscription and additional specific policies can be defined. A blueprint can be configured to decline users to change or delete the resources and policies created by the blueprint.
```

Comment:
upper and lower case mixed for management group and blueprint.
Lets make it upper case everywhere

Reply:
Some words are coming from the Dhall config. I fix these in the PR for the manegemtn group desc.